catalog/task/create-pyxis-image/0.4/create-pyxis-image.yaml

```diff
@@ -44,6 +44,7 @@ spec:
   steps:
     - name: create-pyxis-image
       image:
+        # TODO: Update when the updated release-utils image is available
```

Comment:
Edit the image once https://github.com/redhat-appstudio/release-service-utils/pull/58 is merged.